content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -1,134 +1,112 @@
 one at a time in sequential order. Each must succeed before the next can run.
-When all of the Init Containers have run to completion, Kubernetes initializes
-the Pod and runs the application Containers as usual.
+When all of the init containers have run to completion, Kubelet initializes
+the Pod and runs the application containers as usual.
 
-## What can Init Containers be used for?
+## What can init containers be used for?
 
-Because Init Containers have separate images from app Containers, they
+Because init containers have separate images from app containers, they
 have some advantages for start-up related code:
 
-* They can contain and run utilities that are not desirable to include in the
-  app Container image for security reasons.
-* They can contain utilities or custom code for setup that is not present in an app
-  image. For example, there is no need to make an image `FROM` another image just to use a tool like
-  `sed`, `awk`, `python`, or `dig` during setup.
+* Init containers can contain utilities or custom code for setup that is not present
+  in an app image. For example, there is no need to make an image `FROM` another image
+  just to use a tool like `sed`, `awk`, `python`, or `dig` during setup.
+* Init containers can inclde and run utilities that are kept separate from the app
+  container image for security reasons.
 * The application image builder and deployer roles can work independently without
   the need to jointly build a single app image.
-* They use Linux namespaces so that they have different filesystem views from app Containers.
-  Consequently, they can be given access to Secrets that app Containers are not able to
-  access.
-* They run to completion before any app Containers start, whereas app
-  Containers run in parallel, so Init Containers provide an easy way to block or
-  delay the startup of app Containers until some set of preconditions are met.
+* Init containers can run with a different view of the filesystem than app containers in the
+  same Pod. Consequently, they can be given access to
+  {{< glossary_tooltip text="Secrets" term_id="secret" >}} that app containers are not able
+  to access.
+* Because init containes run to completion before any app containers start, this offers
+  a mechanism to block or delay the startup of app containers until some set of
+  preconditions are met. Once that happens, all the app containers in a Pod can start
+  in parallel.
 
 ### Examples
-Here are some ideas for how to use Init Containers:
-
-* Wait for a service to be created with a shell command like:
+Here are some ideas for how to use init containers:
 
-      for i in {1..100}; do sleep 1; if dig myservice; then exit 0; fi; done; exit 1
+* Wait for a {{< glossary_tooltip text="Service" term_id="service">}} to
+  be created, using a shell one-line command like:
+  ```shell
+  for i in {1..100}; do sleep 1; if dig myservice; then exit 0; fi; done; exit 1
+  ```
```

Comment:
don't think this will work for code inside lists.

Reply:
It seems to look OK in the [deploy preview](https://deploy-preview-14281--kubernetes-io-master-staging.netlify.com/docs/concepts/workloads/pods/init-containers/#examples).

Does that Markdown / rendering need any tweaks?

Reply:
OK